### PR TITLE
Add code to automatically delete Trick'cord Treat messages

### PR DESCRIPTION
**Pull request by navarr**

### CODE REVIEW

File: src/Utility/TricordTreat.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        if (!message.member) {
+            return;
+        }
+
+        if (typeof this.config.trickcordTreatRoom !== 'undefined'
+            && message.channel.id == this.config.trickcordTreatRoom) {
+            return;
+        }
+
+        message.member.is
```
Comment (JamsheedMistri) [inline src/Utility/TricordTreat.js]: What is this line for?

**Review by JamsheedMistri — CHANGES_REQUESTED** [review]
In addition to the line I commented on, "Trick'cord" is spelled inconsistently throughout this PR. Sometimes it's "trickcord" (like in `config/default.js`) and other times it's "tricord" (like the name of the file in `src/Utility` and some of the variables.

File: src/Utility/TricordTreat.js (hunk 1 above)
Comment (navarr) [reply src/Utility/TricordTreat.js]: No idea, tbh.  It's gone now

**Comment by navarr** [comment]
@JamsheedMistri: Good catch.  I've corrected everything to the appropriate spelling.

**Pull request merged**